manual/src/cmds/intf-c.etex

```diff
@@ -740,7 +740,9 @@ numbered from 0 to $\hbox{"string_length"}(v)-1$.
 \var{v}, with type "const char *".
 This pointer is a valid C string: there is a null byte after the last
 byte in the string. However, OCaml strings can contain embedded null bytes,
-which will confuse the usual C functions over strings.
+which will confuse the usual C functions over strings. The function
+"caml_string_is_c_safe("\var{v}")" returns whether the OCaml string \var{v}
```

Comment:
Perhaps I would have used `checks` instead of `returns`.

Reply:
Fixed, thanks.